spec/game_spec.rb

```diff
@@ -0,0 +1,11 @@
+require './lib/game.rb'
+require './app.rb'
+
+describe Rps do
+  it 'shows the game results' do
+    player1 = instance_double('Player', name: 'Nico', move: 'rock' )
+    bot = instance_double('Bot', name: 'Bot', move: 'rock')
```

Comment:
```suggestion
    bot = instance_double('Player', name: 'Bot', move: 'rock')
```
As far as I can see you have no class called `Bot`. In an `instance_double` the string you pass should be the name of the class you're doubling